rotkehlchen/chain/ethereum/modules/curve/pools_cache.py

```diff
@@ -113,6 +115,8 @@ def ensure_curve_tokens_existence(
                         weight=ONE,
                     )],
                     seen=TokenSeenAt(description='Querying curve pools'),
+                    name=ethaddress_to_identifier(token_address),
```

Comment:
Name being the token identifier :thinking: Does it make any sense?

Reply:
Replaced this but is what we do when the token is missing the name field in other places in the code